include/glow/Support/Compiler.h

```diff
@@ -16,6 +16,8 @@
 #ifndef GLOW_SUPPORT_COMPILER_H
 #define GLOW_SUPPORT_COMPILER_H
 
+#include <stdio.h>
```

Comment:
while we are here, GLOW_GNUC_ATLEAST seems to be unused. Let's purge this as well.

Reply:
Pushed another commit in this PR that removes GLOW_GNUC_ATLEAST